tools/gulp-extras.js

```diff
@@ -99,7 +100,24 @@ var checkImports = function() {
     });
 };
 
+var executeCommand = function(command, args, callback) {
+    var process = child_process.spawn(command, args);
+
+    process.stdout.on('data', function(data) {
+        console.log("" + data);
+    });
+
+    process.stderr.on('data', function(data) {
+        console.error("" + data);
+    });
+
+    process.on('exit', function(code) {
```

Comment:
The exit code will take care of reporting errors to gulp via the `callback`, while the error output will be   streamed to the console via `process.stderr`. To my knowledge, there is no `error` event on process.